app/src/main/java/com/framgia/soundclound/screen/detailalbum/DetailAlbumModelView.java

```diff
@@ -22,23 +29,71 @@
 
     public DetailAlbumModelView(Context context, int idAlbum) {
         mContext = context;
-        mDetailAlbumAdapter = new DetailAlbumAdapter();
+        mTracks = AlbumRepository.getInstance(mContext).getAllTrack(idAlbum);
+        mDetailAlbumAdapter = new DetailAlbumAdapter
+                (mTracks);
```

Comment:
ERROR: 'operator new lparen' have incorrect indentation level 16, expected level should be 8.